packages/valory/connections/p2p_libp2p/tests/test_aea_cli.py

```diff
@@ -177,42 +145,3 @@ def test_agent(self):
 
         self.set_logging()
         self.run_aea_cli_test()
-
-
-@libp2p_log_on_failure_all
```

Comment:
Moved to packages/valory/connections/test_libp2p/tests/test_p2p_libp2p/test_aea_cli.py